edk2toollib/uefi/status_codes_test.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+
+from edk2toollib.uefi.status_codes import UefiStatusCode
+
+class TestUefiStatusCodes (unittest.TestCase):
+
+    def test_Hex64ToString (self):
```

Comment:
Since you've touched the file, maybe you can add a few more tests? :)

Reply:
Tests added